src/lib/errors.ts

```diff
@@ -103,6 +103,12 @@ export class ContractViolationError extends TypedError {
 	}
 }
 
+export class ExtLinuxParseError extends TypedError {
+	public constructor(parseError: string) {
```

Comment:
You can omit the constructor when it only calls `super`.

Reply:
fixed!